apps/site/src/app/(main)/apply/sections/ApplicationsClosed/ApplicationsClosed.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+
+function ApplicationsClosed() {
+	return (
+		<div className="rounded-2xl p-10 text-display text-[#432810] bg-white text-center w-8/12 max-[800px]:w-9/12 max-[400px]:w-11/12">
+			<h1 className="text-4xl p-10">
+				<p>Applications for IrvineHacks 2024 were closed on January 14th.</p>
```

Comment:
Nitpick: passive voice unnecessary, and paragraph unnecessary since already `h1`.
```suggestion
				Applications for IrvineHacks 2024 closed on January 14th.
```